Dockerfile.test.dapper

```diff
@@ -2,18 +2,16 @@ FROM golang:1.13.15-alpine3.12
 
 RUN apk -U --no-cache add bash git gcc musl-dev docker curl jq coreutils python2 openssl
 
-ARG DAPPER_HOST_ARCH
-ENV ARCH $DAPPER_HOST_ARCH
+ENV SONOBUOY_VERSION 0.19.0
```

Comment:
do we want to stick with the 0.19.x sonobuoy or match the minor version to the kubernetes under test?

Reply:
My understanding is that sonobuoy itself should be equal to or greater than the kubernetes version. The actual tests use conformance images with tests that are matched to the specific patch release.